.circleci/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: starefossen/ruby-node:2-8
+    steps:
+      - checkout
+      - run: ./go ci_test
```

Comment:
Does bundler ([more info](https://circleci.com/docs/2.0/language-ruby/)) need to run before running this command?

Reply:
`go` takes care of it. Happy to split it out if the clarity would help.

https://github.com/18F/before-you-ship/blob/1c7611d2803407ec2ada9ab619cc6b8e1f39e762/go#L12-L18

Reply:
I've never used a CI system before, so I'm afraid I can't provide any quality feedback on this.

Reply:
If that's the case, I'm for the less code is better approach  👍